Incident Search Updates

ciabhanconnelly opened the pull request:
Added the ability to search for a specific incident ID, and to filter by author with a dropdown.

ciabhanconnelly committed:
Incident Search Updates

Added the ability to search for a specific incident ID, and to filter by author with a dropdown.

limmabean commented:
![image](https://user-images.githubusercontent.com/25267840/107282149-99698e80-6a28-11eb-89af-51b230a7413f.png)

Just removes the incidents I have without me searching anything.

limmabean commented:
![image](https://user-images.githubusercontent.com/25267840/107282795-7b505e00-6a29-11eb-873c-cd59caf8b0ac.png)
It doesn't break my incidents now, but it does not work.

ciabhanconnelly committed:
package lock

package lock is showing up as changed locally for some reason. Maybe this is the issue?

limmabean commented:
![image](https://user-images.githubusercontent.com/25267840/107284047-2f061d80-6a2b-11eb-92fe-e886a2d38b7e.png)
Whenever I get a problem where I don't see any incidents on the incident page on default, I can search by ID but not by user.

ciabhanconnelly committed:
working version

working version of incident search, but the id searched will return an id that shows that id + 1. Fixing this would either change old IDs shown to the user, or break the feature. Trying to find another answer.

ciabhanconnelly committed:
Bugfix

Fixed the issue with idnum either being a vlue off from the number shown to the user, or cause incidents not to play unless their id was explicitly searched.

limmabean reviewed (approved):
Works like a charm now.
